Brokerages/Alpaca/AlpacaBrokerageFactory.cs

```diff
@@ -92,18 +92,13 @@ public override IBrokerage CreateBrokerage(LiveNodePacket job, IAlgorithm algori
                 throw new Exception("Available trading mode: paper/live");
             }
 
-            var handlesMarketData = job.DataQueueHandler.EndsWith("AlpacaBrokerage");
-
-            var brokerage = new AlpacaBrokerage(algorithm.Transactions, 
+            return new AlpacaBrokerage(
+                algorithm.Transactions, 
                 algorithm.Portfolio, 
                 keyId, 
                 secretKey, 
                 tradingMode, 
-                handlesMarketData,
                 Composer.Instance.GetExportedValueByTypeName<IDataAggregator>(Config.Get("data-aggregator", "QuantConnect.Lean.Engine.DataFeeds.AggregationManager")));
-            Composer.Instance.AddPart<IDataQueueHandler>(brokerage);
```

Comment:
Maybe we should initialize the new polygon DQH here if `Alpaca` has been selected as DQH?